networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -43,6 +43,18 @@ def generic_bfs_edges(G, source, neighbors=None, depth_limit=None, sort_neighbor
     depth_limit : int, optional(default=len(G))
         Specify the maximum search depth.
 
+    sort_neighbors : Callable
+
+        .. deprecated:: 3.2
+
+           The sort_neighbors parameter is deprecated and will be removed in
+           version 3.4. A custom (e.g. sorted) ordering of neighbors can be
+           specified with the `neighbors` parameter.
+
+        A function that takes the list of neighbors of a given node as input,
+        and returns an iterator over these neighbors but with a custom
+        ordering.
```

Comment:
Hmm, I was trying to look up the docs in the circle CI build and it looks like `generic_bfs_edges` isn't exposed in the nx namespace? It's not in `__all__` too. Shouldn't we building docs and exposing this on the doc site?

Reply:
Sounds good to me. It should definitely be documented. I'm not sure whether the omission from `__all__` is intentional but I'll add it here on the assumption it's not - please LMK if I'm wrong!